src/exampleTypes/Bool.js

```diff
@@ -1,17 +1,34 @@
 import React from 'react'
-import F from 'futil'
+import _ from 'lodash/fp'
 import { contexturify } from '../utils/hoc'
 
-let Bool = ({ tree, node, theme: { RadioList } }) => (
-  <div className="contexture-bool">
-    <RadioList
-      value={node.value ? 'yes' : 'no'}
-      onChange={value => {
-        tree.mutate(node.path, { value: value === 'yes' })
-      }}
-      options={F.autoLabelOptions(['yes', 'no'])}
-    />
-  </div>
-)
+let getValue = (value, either) =>
+  _.isBoolean(value) ? value : either ? null : false
+
+let Bool = ({
+  tree,
+  node,
+  display = () => ['Yes', 'No', 'Either'],
```

Comment:
this API is different from `facet` - let's have them match and have display be a function that takes the value and returns label

```js
value => _.isNil(value) ? 'Either' : value ? 'Yes' : 'No'
```

You're welcome to export a utility that takes an array:

```js
let generateBoolDisplay: ([yes, no, either]) => value => _.isNil(value) ? either : value ? yes : no
```

But the actual API should follow the convention we have in `facet` and results table of display being a function that takes a value and returns a label. With that said, the issue with relying on array order is that there is no obvious ordering - you'd have to look it up in the docs either way, where as a `value ->label` function follows the convention and doesn't require you to remember an arbitrary order (even if it's more verbose)